src/service-worker/install.ts

```diff
@@ -0,0 +1,16 @@
+/// <reference lib="webworker" />
+import { CURRENT_CACHE_NAME } from './common';
+import { build } from '$service-worker';
+
+export function handleInstall(event: ExtendableEvent): void {
+  event.waitUntil(
+    // Cache known build output and static files
+    // @see <https://kit.svelte.dev/docs/modules#$service-worker-build>
+    caches.open(CURRENT_CACHE_NAME).then((cache) => {
```

Comment:
Nit: Do we have to use `then` here or can we use `await`?

Reply:
Good observation. Async function handlers are probably allowed to be passed to the event listeners, but we currently do not have strict style guidelines for using `await` over `then`. One advantage of `then`, for example, is its ability to chain error handling (via `catch` and `finally`) instead of wrapping multiple lines with `try`/`catch`.